packages/driver/src/cypress/cy.ts

```diff
@@ -327,7 +339,7 @@ export class $Cy extends EventEmitter2 implements ITimeouts, IStability, IAssert
     this.ensureSubjectByType = ensures.ensureSubjectByType
     this.ensureRunnable = ensures.ensureRunnable
 
-    const snapshots = createSnapshots(this.$$, state)
+    const snapshots = createSnapshots(jquery.$$, state)
```

Comment:
```suggestion
    const snapshots = createSnapshots(this.$$, state)
```

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/21137/commits/9a5aecb512e43663d46c6cd2c3d3db1da2b8e26f